test/test_utils.py

```diff
@@ -0,0 +1,173 @@
+import pytest
+from mock import patch, call
+from dateutil import parser
+
+from dafni_cli import utils
+from dafni_cli.model import Model
+from test.fixtures.model_fixtures import get_models_list_fixture
+
+
+@patch("dafni_cli.utils.click")
+class TestProsePrint:
+    """Test class to test the prose_print() functionality"""
+
+    def test_single_paragraph_outputs_correctly(self, mock_click):
```

Comment:
more detailed name please, as this is case where the string is longer than the width, so its auto splitting as opposed to looking for \n